pipeline/build-package-ui.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+jobs:
+    - job: 'build_and_publish_package'
+
+      pool:
+          vmImage: 'macOS-10.14'
+
+      steps:
+          - template: install-node-prerequisites.yaml
+
+          - script: yarn lint:check
```

Comment:
this block including lint, format and copyright checks is repeated on other yaml files too. Does it make sense to extract this to it's own yaml file and the use the `template` directive here and on the other yaml files that use this steps too?

Reply:
Yes, probably. I will do it at least for the two packages yamls.

Reply:
done!